hack/update_catalog.sh

```diff
@@ -0,0 +1,82 @@
+# Helper tool to build the File Based Catalog (FBC) for the operator with semantic versioning
+# Example , if you want to build catalog image for bundle image, set the
+# VERSION variable to 0.0.2 and run the script. The script will build the bundle image and
+# catalog image and push it to the quay.io registry.
+# Pre-requisites: opm, podman, make
+# Usage: ./hack/update_catalog.sh
+
+#!/bin/bash
+
+# Backup files before running script
+backup() {
+  echo "Backing up files"
+  mkdir -p backup
+  cp "${CSV_FILE}" backup/lightspeed-operator.clusterserviceversion.yaml.bak
+  cp "${KUSTOMIZATION_FILE}" backup/kustomization.yaml.bak
+  cp "${CATALOG_FILE}" backup/operator.yaml.bak
+  cp "${OLS_CONFIG_FILE}" backup/ols.openshift.io_olsconfigs.yaml.bak
+}
+
+
+# Reverse backup files
+restore() {
+  echo "Restoring files"
+  cp backup/lightspeed-operator.clusterserviceversion.yaml.bak "${CSV_FILE}"
+  cp backup/kustomization.yaml.bak "${KUSTOMIZATION_FILE}"
+  cp backup/operator.yaml.bak "${CATALOG_FILE}"
+  cp backup/ols.openshift.io_olsconfigs.yaml.bak "${OLS_CONFIG_FILE}"
+  rm -rf backup
+}
+
+trap restore EXIT
+
+set -euo pipefail
+DEFAULT_PLATFORM="linux/amd64"
+export QUAY_USER=""  # Set quay user for personal builds
+VERSION="0.0.1"    # Set the bundle version
+QUAY_USER="${QUAY_USER:-openshift}"
```

Comment:
The QUAY_USER is always `openshift` after this line because line 35 has set it to empty. 
Are we going to use `openshift` as QUAY_USER for always? If so we can replace line 35 and 37 by `QUAY_USER=openshift`

Reply:
I have left it to have User id , so that I can test the image push to my repo - I dont have access to push to openshift repo @raptorsun